src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+---
+title: Issues API using Nerdgraph
+metaDescription: "Set capabilities and entitlements in Detection and AI"
+redirects: 
+---
+
+This doc is a place for you and your team to learn how to query your data for issues and incidents and then configure features using our NerdGraph API.
+
+Our two main data types are `Issue Type` and `Incident Type`. 
```

Comment:
what do we mean by 'data types' here? is this simply referring to the domain of alerts and AI 'issues and incidents'? I'm a bit lost as to what the context is, and maybe we can say somethign like 'In NerdGraph, issues and incidents are represented by these two data types: ..."

Also below, I think we should be more specific. Where it says 'learn more about New Relic's data types...', they're learning about the two specific data types (I think) , so we should specific what the follow tables are about

Reply:
Waiting on response from AI team. Will create a fast follow when I hear back.

Reply:
@neta-signifai would you mind clarifying what `data types` mean in the context of this document?